### ISSUE TITLE
 vue3-ThreeJs_minecraft-building

### ISSUE THREAD

**Issue by Y3ssirski** [seungmin-han/vue3-ThreeJs_minecraft-Building-Portfolio#1]
hey bro Is it the model imported in Minecraft?

**Comment by seungmin-han** [comment]
yes. i build it

**Comment by seungmin-han** [comment]
> hey bro Is it the model imported in Minecraft?

yes. i build it

**Comment by Y3ssirski** [comment]
Which website is Minecraft? I can't find the relevant one ：）

**Comment by seungmin-han** [comment]
> 

this buildings in my minecraft world with little tiles mod. Due to computer specifications, only some buildings were imported as models.

**Comment by Y3ssirski** [comment]
Build a building in my minecraft world, export the file and put it in the program ？

**Comment by seungmin-han** [comment]
> Build a building in my minecraft world, export the file and put it in the program ？

nope, export model to mod. mod name is 'little tiles 3D Importer&Exporter'    : )

**Comment by Y3ssirski** [comment]
thank u :D

**Comment by Y3ssirski** [comment]
Is mod in the vue3 program?: D

**Comment by seungmin-han** [comment]
> Is mod in the vue3 program?: D

'little tiles 3D Importer & Exporter' mod is minecraft mod with forge. 

the minecraft & forge version is 1.12.2.

ref:

https://files.minecraftforge.net/net/minecraftforge/forge/index_1.12.2.html
https://www.curseforge.com/minecraft/mc-mods/littletiles
https://www.curseforge.com/minecraft/mc-mods/littletiles-3d-importer-exporter

**Comment by Y3ssirski** [comment]
감사합니다.  XDDD
